main_logic/tts_client.py

```
@@ -193,17 +194,36 @@ async def receive_messages_initial():
             loop = asyncio.get_running_loop()
             while True:
                 try:
                     sid, tts_text = await loop.run_in_executor(None, request_queue.get)
                 except Exception:
                     break
-                
+            try:
+                while True:
+                    try:
+                        sid, tts_text = await loop.run_in_executor(None, request_queue.get)
+                    except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)